schema.sql

```diff
@@ -63,10 +65,10 @@ CREATE SCHEMA web3storage
     aggregate_id bigint NOT NULL,
     storage_provider TEXT,
     deal_id bigint UNIQUE NOT NULL,
-    activation timestamp(6) without time zone,
-    renewel timestamp(6) without time zone,
+    activation timestamp with time zone,
+    renewal timestamp with time zone,
```

Comment:
Typo in "renewal" and also switch to _with_ time zone for consistency. Again, I'm not sure if that was intentional?